src/feed/feed.service.ts

```
@@ -44,6 +48,22 @@ export class FeedService {
     };
   }
 
+  async getWeeklyNewPostsCount(): Promise<Array<{ week: string; count: number }>> {
```

ESLint

[error] Replace `Array<{·week:·string;·count:·number·}>` with `⏎····Array<{·week:·string;·count:·number·}>⏎··`

(prettier/prettier)